docs/csharp/reference-semantics-with-value-types.md

```diff
@@ -111,7 +134,7 @@ that you are returning a reference to existing data, but not allowing
 modification. 
 
 The compiler enforces that the caller cannot modify the reference. Attempts
-to assign to the value directly generate a compile-time error. However, the compiler cannot know if any member method modifies the state of the struct.
+to assign the value directly generate a compile-time error. However, the compiler cannot know if any member method modifies the state of the struct.
```

Comment:
This contradicts line 90 and following.

Reply:
PR #4710 address this (based on my review comments).

Either way, I'll update this paragraph after merging that PR.